### PR TITLE
Time stamps were not being set to now if they were blank

### LINKED ISSUE #1542: Writing points without a timestamp should assign the server's current timestamp
At the moment, it appears that these points are accepted for writes, but magically vanish. For example:

```
{
    "database": "foo",
    "retentionPolicy": "",
    "points": [
        {
            "name": "cpu_load_short",
            "values": {
                "value": 1
            }
        }
    ]
}
```

Won't return any values when you issue `SELECT value FROM cpu_load_short` afterwards.

### PR DESCRIPTION
**Pull request by corylanou**
Fixes  #1542

### CODE REVIEW

File: httpd/handler_test.go [hunk 1]
```diff
@@ -1094,6 +1094,56 @@ func TestHandler_serveWriteSeries_noDatabaseSpecified(t *testing.T) {
...
+
+	now := time.Now()
+
+	status, _ := MustHTTP("POST", s.URL+`/write`, nil, nil, `{"database" : "foo", "retentionPolicy" : "bar", "points": [{"name": "cpu", "tags": {"host": "server01"},"values": {"value": 100}}]}`)
+
+	if status != http.StatusOK {
+		t.Fatalf("unexpected status: %d", status)
+	}
+
+	query := map[string]string{"db": "foo", "q": "select value from cpu"}
```
Comment (otoolep) [inline httpd/handler_test.go]: You could tighten this up by SELECT data with a time greater than `now`.

**Comment by otoolep** [comment]
I think you could make your test a little nicer, but that is up to you. :-)

Change looks good. Ship it.

**Comment by corylanou** [comment]
@otoolep I'm open to making the test nicer, what were your thoughts?

**Comment by otoolep** [comment]
Well, if you use `WHERE time > now` (replacing `now` with the variable you set) you wouldn't have to check for a zero timestamp in the returned results. Assuming `WHERE` worked properly. 

It's not a huge change, I wouldn't worry too much about it -- it might actually be more confusing.

**Comment by toddboom** [comment]
:clap:

**Pull request merged**